spikeinterface/core/core_tools.py

```diff
@@ -80,53 +80,66 @@ def write_python(path, dict):
 
 
 def check_json(d):
+    dc = deepcopy(d)
     # quick hack to ensure json writable
     for k, v in d.items():
+        # take care of keys first
+        if isinstance(k, np.int64):
+            del dc[k]
+            dc[int(k)] = v
+        if isinstance(k, np.float64):
+            del dc[k]
+            dc[float(k)] = v
```

Comment:
This is needed to avoid keys to be non serializable (e.g. for toy_example, where channel ids and unit ids are int64). The use case is for saving the sparsity dictionary as params